cluster/olm/ceph/generate-rook-csv-templates.sh

```diff
@@ -32,7 +32,7 @@ function generate_template() {
     mv $tmp_csv_gen_file $csv_template_file
 
     # replace the placeholder with the templated value
-    $SED "s/9999.9999.9999/{{.RookOperatorCsvVersion}}/g" $csv_template_file
+    $SED_IN_PLACE "s/9999.9999.9999/{{.RookOperatorCsvVersion}}/g" $csv_template_file
```

Comment:
Why is this instance of `sed` the only one to not use `-e`? Why is `-e` necessary? Can we include it in the `SED_IN_PLACE` script if so?

Reply:
It's necessary on Mac but I don't think it is on Linux. I thought about putting it into the `sed-in-place` script, but it would break things if a user wanted to also use a different argument like `-f`. The old `SED_CMD` included the `-e` arg, so it seems fine to include it in the script. I'll make that change.